x-pack/plugins/lens/public/editor_frame_service/editor_frame/save.ts

```diff
@@ -69,6 +69,7 @@ export function getSavedObjectFormat({
         visualization: state.visualization.state,
         query: framePublicAPI.query,
         filters: persistableFilters,
+        activeData: state.activeData,
```

Comment:
This would add `activeData` to the saved object as well (the thing stored in Elasticsearch). Please don't do that, but just pass it up to the app and keep it in local state there (like the `isSaveable` flag and `filterableIndexPatterns ` in line 76,77)